tests/src/JIT/SIMD/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required (VERSION 2.6)
+project (Vector3TestNative)
+include_directories(${INC_PLATFORM_DIR})
+set(SOURCES Vector3TestNative.cpp )
+
+# add the executable
+add_library (Vector3TestNative SHARED ${SOURCES})
+
+# add the install targets
+install (TARGETS Vector3TestNative DESTINATION bin)
```

Comment:
I copied this file from other Interop testcase and just changed the names. I got the native so library via build.sh and it worked well.